projects/go-lib/src/lib/components/go-table/go-table.component.scss

```diff
@@ -213,3 +213,11 @@
     text-align: center;
   }
 }
+
+.go-table__body-row--expanded {
+  border-bottom: none;
```

Comment:
What do you think about changing this to change the opacity of the color of the bottom border? That way we can animate the color and basically fade out the border.